starfish/starfish-core/src/mutate/edge.rs

```diff
@@ -393,14 +396,49 @@ impl Mutate {
             map_id_to_ancestors.insert(root_node.name, ancestors);
         }
 
+        if map_id_to_ancestors.is_empty() {
+            return Ok(());
+        }
+
         // map_id_to_ancestors is ready; the sizes of the sets in its values are the compound in_conn
         let cols = [
             NodeIden::Name.into_iden(),
             Alias::new(&format!("{}_{}", relation_name, col_name)).into_iden(),
         ];
+        
+        if cfg!(feature = "sqlx-sqlite") {
```

Comment:
We should check if the `DbBackend` of `DbConn` was SQLite or not, instead of feature guarded

Reply:
Oh it didn't come to my mind